guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md

```diff
@@ -143,6 +143,61 @@ To edit the WAF error page:
 
    -  On the Cache Management page, click **Flush Magento Cache**.
 
+## Display Magento error report number
+
+By default, Fastly hides all Magento errors behind the _503 Service Unavailable_ error. To display the Magento error log report number so that you can find and review the error details in the logs, open the website omitting Fastly using these steps:
+
+1. Retrieve the IP address of your store:
+
+   -  Pro plan users, Staging and Production environments:
+
+      ```bash
+      nslookup {your_project_id}.ent.magento.cloud
+      ```
+
+   -  Pro plan users, Integration environment; Starter plan users, all environments:
```

Comment:
Consistency.
In many other Cloud guide places, we say "For Pro Staging and Production environments..." and "For Started environments and Pro Integration environment..."